### PR TITLE
DOCSUP-8654: Update the unhex function docs

### PR DESCRIPTION
**Pull request by adevyatova**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation for #23229

### CODE REVIEW

File: docs/ru/sql-reference/functions/encoding-functions.md [hunk 1]
```diff
@@ -153,8 +153,43 @@ Result:
 
 ## unhex(str) {#unhexstr}
 
-Accepts a string containing any number of hexadecimal digits, and returns a string containing the corresponding bytes. Supports both uppercase and lowercase letters A-F. The number of hexadecimal digits does not have to be even. If it is odd, the last digit is interpreted as the least significant half of the 00-0F byte. If the argument string contains anything other than hexadecimal digits, some implementation-defined result is returned (an exception isn’t thrown).
-If you want to convert the result to a number, you can use the ‘reverse’ and ‘reinterpretAsType’ functions.
+Выполняет операцию обратную [hex](#hex). Функция интерпретирует каждую пару шестнадцатеричных цифр аргумента как число и преобразует его в символ.
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/encoding-functions.md:156]:
```suggestion
Выполняет операцию, обратную [hex](#hex). Функция интерпретирует каждую пару шестнадцатеричных цифр аргумента как число и преобразует его в символ.
```

File: docs/en/sql-reference/functions/encoding-functions.md [hunk 2]
```diff
@@ -151,10 +149,62 @@ Result:
 └──────────────────┘
 ```
 
-## unhex(str) {#unhexstr}
+## unhex {#unhexstr}
+
+Performs the opposite operation of [hex](#hex). It interprets each pair of hexadecimal digits (in the argument) as a number and converts it to the byte represented by the number. The return value is a binary string.
```
Comment (CurtizJ) [inline docs/en/sql-reference/functions/encoding-functions.md:154]: The term `binary string` usually refers to a string, that consists of 0-s and 1-s. But also it is used to define blobs in  SQL (types `BINARY` and `VARBINARY` in MySQL. Maybe add a clarification, that it is related to blob in that case.

Comment (CurtizJ) [inline docs/en/sql-reference/functions/encoding-functions.md:154]:
```suggestion
Performs the opposite operation of [hex](#hex). It interprets each pair of hexadecimal digits (in the argument) as a number and converts it to the byte represented by the number. The return value is a binary string (BLOB).
```

File: docs/en/sql-reference/functions/encoding-functions.md [hunk 3]
```diff
@@ -151,10 +149,62 @@ Result:
...
-If you want to convert the result to a number, you can use the ‘reverse’ and ‘reinterpretAsType’ functions.
+**Arguments**
+
+-   `arg` — A string containing any number of hexadecimal digits. Type: [String](../../sql-reference/data-types/string.md).
+
+Supports both uppercase and lowercase letters `A-F`. The number of hexadecimal digits does not have to be even. If it is odd, the last digit is interpreted as the least significant half of the `00-0F` byte. If the argument string contains anything other than hexadecimal digits, some implementation-defined result is returned (an exception isn’t thrown). For a numeric argument the inverse of hex(N) is not performed by unhex().
+
+**Returned value**
+
+-   A binary string.
```
Comment (CurtizJ) [inline docs/en/sql-reference/functions/encoding-functions.md:177]:
```suggestion
-   A binary string (BLOB).
```

File: docs/ru/sql-reference/functions/encoding-functions.md [hunk 4]
```diff
@@ -153,8 +153,60 @@ Result:
...
+
+**Аргументы**
+
+-   `arg` — Строка, содержащая любое количество шестнадцатеричных цифр. Тип: [String](../../sql-reference/data-types/string.md).
+
+Поддерживаются как прописные, так и строчные буквы `A-F`. Количество шестнадцатеричных цифр не обязательно должно быть четным. Если оно нечетное, последняя цифра интерпретируется как наименее значимая половина байта `00-0F`. Если строка аргумента содержит что-либо, кроме шестнадцатеричных цифр, возвращается некоторый результат, определенный реализацией (исключение не создается).
+
+**Возвращаемое значение**
+
+-   Бинарная строка.
```
Comment (CurtizJ) [inline docs/ru/sql-reference/functions/encoding-functions.md:179]:
```suggestion
-   Бинарная строка (BLOB).
```

File: docs/ru/sql-reference/functions/encoding-functions.md [hunk 5]
```diff
@@ -153,8 +153,60 @@ Result:
 
 ## unhex(str) {#unhexstr}
 
-Accepts a string containing any number of hexadecimal digits, and returns a string containing the corresponding bytes. Supports both uppercase and lowercase letters A-F. The number of hexadecimal digits does not have to be even. If it is odd, the last digit is interpreted as the least significant half of the 00-0F byte. If the argument string contains anything other than hexadecimal digits, some implementation-defined result is returned (an exception isn’t thrown).
-If you want to convert the result to a number, you can use the ‘reverse’ and ‘reinterpretAsType’ functions.
+Выполняет операцию, обратную [hex](#hex). Функция интерпретирует каждую пару шестнадцатеричных цифр аргумента как число и преобразует его в символ. Возвращаемое значение представляет собой двоичную строку.
```
Comment (CurtizJ) [inline docs/ru/sql-reference/functions/encoding-functions.md:156]:
```suggestion
Выполняет операцию, обратную [hex](#hex). Функция интерпретирует каждую пару шестнадцатеричных цифр аргумента как число и преобразует его в символ. Возвращаемое значение представляет собой двоичную строку (BLOB).
```

**Review by CurtizJ — APPROVED** [review]

**Pull request merged**